packages/app-project/src/screens/ClassifyPage/components/ClassifierWrapper/ClassifierWrapper.js

```diff
@@ -53,20 +54,22 @@ export default function ClassifierWrapper({
   const ownerSlug = router?.query.owner
   const projectSlug = router?.query.project
 
-  /* Only increment stats on the classify page if the subject is not retired or not already seen by current user */
-  const incrementStats = yourStats?.increment
+  /*
+    Increment user stats on every classification submitted.
+    Add the recently classified subject to the user's Recents.
+  */
```

Comment:
ERAS counts all classifications as a +1 regardless of subject status as 'already seen' or 'retired'. Therefore, we want to increment user stats after every submitted classification.